tests/unit/commands/init-test.js

```diff
@@ -91,12 +93,23 @@ describe('init command', function() {
 
 
   it('Uses process.cwd if no package is found when calling installBlueprint', function() {
+    // change the working dir so `process.cwd` can't be a invalid path for base directories 
+    // named `ember-cli`.
+
+    var tmpDir = os.tmpdir();
+    var workingDir = tmpDir + '/ember-cli-test-project';
+    var currentWorkingDir = process.cwd();
+
+    fs.mkdirSync(workingDir);
+    process.chdir(workingDir);
```

Comment:
im not sure if this works correctly in windows. If i recall something does go side-ways here.